mono/mini/method-to-ir.c

```diff
@@ -8937,6 +8937,11 @@ mono_method_to_ir (MonoCompile *cfg, MonoMethod *method, MonoBasicBlock *start_b
 		case MONO_CEE_STIND_I: {
 			sp -= 2;
 
+			// Workaround weak ARM memory model.
+#if defined (HOST_ARM) || defined (HOST_ARM64)
```

Comment:
shouldn't this be `#if defined (TARGET_ARM) || defined (TARGET_ARM64)`?

Reply:
Yes.